QUEST_JOBSTEP.tsv

```diff
@@ -2480,7 +2480,7 @@ QUEST_JOBSTEP_20150323_002479	{memo X}Go meet the Pardoner Master at the outer e
 QUEST_JOBSTEP_20150323_002480	{memo X}The Pardoner Master wants you to retrieve the money of the lost indulgence for the assignment. Retrieve the lost money at the region near Aqueduct.
 QUEST_JOBSTEP_20150323_002481	Defeat Jonna Jolem that suddenly appeared
 QUEST_JOBSTEP_20150323_002482	{memo X}The Oracle Master is looking for you at Awaiting Stone Bridge.
-QUEST_JOBSTEP_20150323_002483	Obtain Pawndel's Soul Essence and Pawnd's Spirit Essence at the 1st floor of Tenet Cathedral.{nl}
+QUEST_JOBSTEP_20150323_002483	Obtain Pawndel's Soul Essence and Pawnd's Spirit Essence at the 1st floor of Tenet Chapel.{nl}
```

Comment:
Pawndel's Soul Essence -> Pondell's Soul Essence
Pawnd's Spirit Essence -> Pond's Soul Essence
1st floor of Tenet Chapel -> Tenet Chapel 1F